query.go

```diff
@@ -96,6 +98,9 @@ func (q *Query) StopTime() time.Time {
 // Set the start time of the query
 func (q *Query) SetStartTime(s time.Time) {
 	q.startTL.Val = s
+	if q.alignGroup && q.groupByOffsetDL != nil {
+		q.groupByOffsetDL.Val = s.Sub(time.Unix(0, 0))
```

Comment:
This produces queries like `... GROUP BY time(1m, 1484002977413326u)`. Can you perform the modulo so that the offset is always in the range `[-1m, 1m]` when the interval is 1m?

That should make debugging easier for an end user.